src/pages/api/profile/user-profile/index.ts

```diff
@@ -73,7 +73,7 @@ const patchHandler: Handler = async (request, response) => {
       const { userProfileClient } = await wrappedProfileClient(request, span);
 
       const { data: updatedUserProfile } = await userProfileClient
-        .patchProfile(request.body.userProfilePayload)
+        .patchProfile(request.body)
```

Comment:
Since the `userProfilePayload` has been removed, I also removed it here (of course).
The profile is now directly the payload.